<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Mayank" name="Command line test" tests="2" failures="1" timestamp="1 Jul 2018 14:03:48 GMT" time="0.176" errors="0">
  <testcase name="Demo" time="0.044" classname="Banking.AppTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: 
	at Banking.AppTest.Demo(AppTest.java:22)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase name="SecondTest" time="0.0" classname="Banking.AppTest"/>
</testsuite> <!-- Command line test -->
